include/semver/semver.hpp

```diff
@@ -31,17 +31,29 @@ SOFTWARE.
 
 namespace semver
 {
+
+    using build_identifier_part = uint64_t;
```

Comment:
As it's used for the type of each numeric part of a version, and not just for the build identifier's numeric part, could we name it `numeric_part`?

Reply:
> @katzoded Thanks for the contribution! I have some comments regarding the naming, but after fixing that, I'll approve.

@z4kn4fein ,
You are welcome, I addressed the required comments.